<?xml version="1.0" encoding="UTF-8" ?>
<launch>
<!--    <include file="$(find rr_avc)/launch/mapper.launch"/>-->
    <node name="follower" pkg="rr_common" type="follower" output="screen">
        <param name="MIN_FRONT_VISION" value="0.1"/>
        <param name="MAX_FRONT_VISION" value="10"/>
        <param name="GOAL_DIST" value="1.3"/>
        <param name="GOAL_MARGIN_OF_ERR" value="0.2"/>
        <param name="MIN_SIDE_VISION" value="-1"/>
        <param name="MAX_SIDE_VISION" value="1"/>
        <param name="FOLLOWER_SPEED" value="0.2"/>
        <param name="INPUT_CLOUD_TOPIC" value="/scan/pointcloud"/>
    </node>
    <node name="demo_pid_controller" pkg="pid" type="controller" >
        <param name="Kp" value="5.0" />
        <param name="Ki" value="0.0" />
        <param name="Kd" value="0.1" />
        <param name="upper_limit" value="1" />
        <param name="lower_limit" value="-1" />
        <param name="windup_limit" value="10" />
<!--        <param name="max_loop_frequency" value="100.0" />-->
<!--        <param name="min_loop_frequency" value="100.0" />-->
        <param name="topic_from_controller" value="/control_effort" />
        <param name="topic_from_plant" value="/state" />
        <param name="setpoint_topic" value="/setpoint" />
    </node>
</launch>
